include/proj/internal/esri_projection_mappings.hpp

```diff
@@ -616,6 +616,24 @@ static const ESRIParamMapping paramsESRI_Gnomonic[] = {
      EPSG_CODE_PARAMETER_LATITUDE_OF_NATURAL_ORIGIN, 0.0},
     {nullptr, nullptr, 0, 0.0}};
 
+static const ESRIParamMapping paramsESRI_Times[] = {
+    {"False_Easting", EPSG_NAME_PARAMETER_FALSE_EASTING,
+     EPSG_CODE_PARAMETER_FALSE_EASTING, 0.0},
+    {"False_Northing", EPSG_NAME_PARAMETER_FALSE_NORTHING,
+     EPSG_CODE_PARAMETER_FALSE_NORTHING, 0.0},
+    {"Central_Meridian", EPSG_NAME_PARAMETER_LONGITUDE_OF_NATURAL_ORIGIN,
+     EPSG_CODE_PARAMETER_LONGITUDE_OF_NATURAL_ORIGIN, 0.0},
+    {nullptr, nullptr, 0, 0.0}};
+
+static const ESRIParamMapping paramsESRI_Vertical_Near_Side_Perspective[] = {
```

Comment:
incomplete mapping. Missing Height and Latitude_Of_Center parameters. see https://github.com/Esri/projection-engine-db-doc/blob/master/other/proj4-esri.txt#L396